app/Http/Kernel.php

```diff
@@ -16,7 +16,7 @@ class Kernel extends HttpKernel
     protected $middleware = [
         // \App\Http\Middleware\TrustHosts::class,
         \App\Http\Middleware\TrustProxies::class,
-        \Fruitcake\Cors\HandleCors::class,
+        \Illuminate\Http\Middleware\HandleCors::class,
```

Comment:
Fruitcake\Cors\HandleCors is built into the framework now https://laravel-news.com/laravel-9-2-0